src/main/java/bank/hm33/PersonOperationService.java

```diff
@@ -11,18 +12,24 @@
 import java.util.concurrent.Executors;
 
 @Slf4j
+@Setter
 @AllArgsConstructor
 @NoArgsConstructor
+
 public class PersonOperationService {
 
     private CurrencyConverter converter;
```

Comment:
```suggestion
    private final CurrencyConverter converter;
```
final - щоб не забувати ініціалізовувати через конструктор (ну і семантично це вірно, оскільки не треба змінювати)